CRM/Contribute/Form/ContributionView.php

```diff
@@ -152,7 +152,7 @@ public function preProcess() {
 
     $lineItems = array();
     if ($id) {
-      $lineItem = CRM_Price_BAO_LineItem::getLineItems($id, 'contribution', 1, TRUE, TRUE);
+      $lineItem = CRM_Price_BAO_LineItem::getLineItemsByContributionID($id, TRUE);
```

Comment:
I propose that we can avoid passing the 'true with mysterious implications' by this commit https://github.com/civicrm/civicrm-core/pull/9327/commits/59f7b38eb943851563395eb57c386aa19c239443